OpenRA.Mods.Common/Effects/Bullet.cs

```diff
@@ -145,8 +145,12 @@ public void Tick(World world)
 			if (info.ContrailLength > 0)
 				contrail.Update(pos);
 
-			if (ticks++ >= length || (info.Blockable && world.ActorMap
```

Comment:
This shouldn't change anything performance wise because the ActorMap query is already skipped if `ticks++ >= length` or `info.Blockable == false`.